homeassistant/components/hive/light.py

```diff
@@ -77,7 +77,13 @@ async def async_turn_on(self, **kwargs: Any) -> None:
             get_new_color = kwargs[ATTR_HS_COLOR]
             hue = int(get_new_color[0])
             saturation = int(get_new_color[1])
-            new_color = (hue, saturation, 100)
+            brightness = int(
+                round(
+                    ((self.brightness if self.brightness is not None else 5) / 255)
+                    * 100
+                )
+            )
+            new_color = (hue, saturation, brightness)
```

Comment:
the `hive.light.turnOn` call below passes in the brightness already as well.

I feel like we are mixing up two different types of brightness here?

../Frenck